internal/commands/sbommonitor/sbommonitor.go

```diff
@@ -26,6 +30,33 @@ func RegisterWorkflows(e workflow.Engine) error {
 	return nil
 }
 
+func loadPolicyFile(policyPath, sbomFilePath string) ([]byte, error) {
+	policy := []byte("\n")
+	var policyFilePath string
+	if policyPath != "" {
+		policyFilePath = policyPath
+	} else {
+		policyFilePath = filepath.Join(filepath.Dir(sbomFilePath), ".snyk")
```

Comment:
`sbomFilePath` is unsanitized, it's effectively whatever the user sets on `--file`. how safe is it to use here? do we need to do an `os.Stat` first, either here or in the caller of the function?